### ISSUE TITLE
bindKernelInputs error

### ISSUE THREAD

**Issue by naoyam** [csarofeen/pytorch#547]
## 🐛 Bug

bindKernelInputs fails at `root_domain[dim]->extent()` when the IterDomain is parallelized.

```
C++ exception with description "active_gpu_lower != nullptr INTERNAL ASSERT FAILED at "../torch/csrc/jit/codegen/cuda/lower2device.cpp":300, please report a bug to PyTorch.
Exception raised from current at ../torch/csrc/jit/codegen/cuda/lower2device.cpp:300 (most recent call first):
frame #0: <unknown function> + 0xa5a26 (0x7fb72b1d2a26 in /home/nmaruyama/pytorch/debug1/build/lib/libc10.so)
frame #1: std::function<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > ()>::operator()() const + 0x54 (0x7fb7374ae904 in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cpu.so)
frame #2: c10::Error::Error(c10::SourceLocation, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) + 0x57 (0x7fb72b1d1e47 in /home/nmaruyama/pytorch/debug1/build/lib/libc10.so)
frame #3: torch::jit::fuser::cuda::GpuLower::current() + 0xa4 (0x7fb7408d6654 in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cuda.so)
frame #4: torch::jit::fuser::cuda::kir::NamedScalar::getParallelDim(torch::jit::fuser::cuda::ParallelType) + 0x1c (0x7fb74087d02c in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cuda.so)
frame #5: torch::jit::fuser::cuda::kir::IterDomain::extent() const + 0x86 (0x7fb74087dc96 in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cuda.so)
frame #6: <unknown function> + 0x4bd0d48 (0x7fb7407a8d48 in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cuda.so)
frame #7: torch::jit::fuser::cuda::FusionExecutor::runFusion(c10::ArrayRef<c10::IValue> const&, std::vector<at::Tensor, std::allocator<at::Tensor> > const&, torch::jit::fuser::cuda::LaunchParams const&, c10::optional<unsigned long> const&) + 0xa9f (0x7fb7407758df in /home/nmaruyama/pytorch/debug1/build/lib/libtorch_cuda.so)
```

## To Reproduce

Run this fusion:

```
  auto tv0 = makeSymbolicTensor(1);
  auto tv1 = makeSymbolicTensor(2);
  auto tv2 = broadcast(tv0, {false, true});
  auto tv3 = add(tv1, tv2);
  fusion.addInput(tv0);
  fusion.addInput(tv1);
  fusion.addOutput(tv3);
  tv2->axis(0)->parallelize(ParallelType::TIDx);
  tv3->axis(0)->parallelize(ParallelType::TIDx);
```

`broadcast` reuses the IterDomains of the input tensor for its output tensor, so setting ParallelType of a `tv2` IterDomain indirectly means `tv0` gets the same ParallelType.

**Comment by naoyam** [comment]
#552 will fix this.

**Comment by naoyam** [comment]
Fixed by #552
